vignettes/Tutorial.Rmd

```diff
@@ -75,13 +73,14 @@ spelled as the program will not work if this is not the case.
 ## Load the R package
 library(kwb.heatsine)
 
-path_csv_sw <- kwb.heatsine::extdata_file("temperature_surface-water_TEGsee-mikrosieb.csv")
-path_csv_gw <- kwb.heatsine::extdata_file("temperature_groundwater_TEG343.csv")
-
-
-data_sw <- kwb.heatsine::load_temperature_from_csv(path_csv_sw)
-data_gw <- kwb.heatsine::load_temperature_from_csv(path_csv_gw)
+load_temp <- function(base_name) {
```

Comment:
I think this is one function too much...I want to show the function `load_temperature_from_csv()` which is now masked by l`oad_temp()`